src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -2284,23 +2309,23 @@ void func_80833C3C(Player* this) {
     this->unk_870 = this->unk_874 = 0.0f;
 }
 
-s32 func_80833C50(Player* this, s32 item) {
+s32 Player_ItemIsInUse(Player* this, s32 item) {
     if ((item < ITEM_NONE_FE) && (Player_ItemToItemAction(item) == this->itemAction)) {
         return 1;
     } else {
         return 0;
```

Comment:
Could make `Player_ItemIsInUse` and `Player_ItemIsItemAction` return `true`/`false`